commit ac56d8086cbe27ed74bb708fd9cca3c531f4eb30
Author: Richard Imaoka <richard.e.imaoka@gmail.com>
Date:   Sun May 8 08:19:52 2022 +0900

    server necessary files to start
    
    add index.ts, schema.gql, data.json
    npm set-script server-start "ts-node-dev --watch schema.gql,data.json --respawn index.ts"

diff --git a/server/data.json b/server/data.json
new file mode 100644
index 0000000..0967ef4
--- /dev/null
+++ b/server/data.json
@@ -0,0 +1 @@
+{}
diff --git a/server/index.ts b/server/index.ts
new file mode 100644
index 0000000..78c0639
--- /dev/null
+++ b/server/index.ts
@@ -0,0 +1,16 @@
+import { ApolloServer, gql } from "apollo-server";
+import * as fs from "fs";
+
+const typeDefs = gql`
+  ${fs.readFileSync(__dirname.concat("/schema.gql"), "utf8")}
+`;
+
+const server = new ApolloServer({
+  typeDefs,
+  mocks: true,
+});
+
+// The `listen` method launches a web server.
+server.listen().then(({ url }) => {
+  console.log(`ðŸš€  Server ready at ${url}`);
+});
diff --git a/server/package.json b/server/package.json
index a1d5621..f7f8795 100644
--- a/server/package.json
+++ b/server/package.json
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "server-start": "ts-node-dev --watch schema.gql,data.json --respawn index.ts"
   },
   "keywords": [],
   "author": "",
diff --git a/server/schema.gql b/server/schema.gql
new file mode 100644
index 0000000..6ae991f
--- /dev/null
+++ b/server/schema.gql
@@ -0,0 +1,3 @@
+type Query {
+  hello: String
+}
